internal/rpm/rpm.go

```diff
@@ -8,6 +8,8 @@ import (
 	"path/filepath"
 
 	rpmdb "github.com/knqyf263/go-rpmdb/pkg"
+	// This pulls in the sqlite dependency
+	_ "github.com/glebarez/go-sqlite"
```

Comment:
Isn't it kind of weird that _we_ have to initialize go-sqlite, vs. the rpmdb lib doing so? Or is this one of those things where "we choose"?

Reply:
Pretty sure this is a `we choose` type deal since stdlib only gives the interface for DB's and then the caller chooses the implementation.